zappa/core.py

```diff
@@ -2919,7 +2919,17 @@ def get_scheduled_event_name(self, event, function, lambda_name, index=0):
         """
         Returns an AWS-valid CloudWatch rule name using a digest of the event name, lambda name, and function.
         This allows support for rule names that may be longer than the 64 char limit.
+
+        function pattern: '^[._A-Za-z0-9]{,63}$'
```

Comment:
Can we update this to `{0,63}` for clarity?

Reply:
Thank you! fixed